helix-term/src/ui/picker.rs

```diff
@@ -46,7 +46,7 @@ pub struct FilePicker<T> {
 }
 
 pub enum CachedPreview {
-    Document(Document),
+    Document(Box<Document>),
```

Comment:
Using `Box<Document>` rather than `Document` is trying to address this warning: https://rust-lang.github.io/rust-clippy/master/index.html#large_enum_variant

Reply:
Huh, that's a good catch.